internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -539,19 +532,6 @@ func (r *AddOnDefinitionResource) Update(ctx context.Context, req resource.Updat
 		modelBody.ReadmeMarkdownTemplate = readmeBytes
 	}
 
-	if !plan.TerraformCloudWorkspace.IsNull() && !plan.TerraformCloudWorkspace.IsUnknown() {
```

Comment:
This was accidentally included in a merged PR so I am removing it so it's back to how it used to function.